internal/chart/helm_client.go

```diff
@@ -51,9 +61,75 @@ func getActionConfig(namespace string) (*action.Configuration, error) {
 	return actionConfig, nil
 }
 
-// Option to perform additional configuration of action.Install before running a chart installation.
+// InstallOption to perform additional configuration of action.Install before running a chart installation.
 type InstallOption func(install *action.Install)
 
+type postRender struct {
+	client.Client
```

Comment:
why do we want to embed client.Client? 
how about 
```golang
type postRender struct {
	cli client.Client
	namespace string
}

```

Reply:
Changed.